frontend/app/src/store/balances/types.ts

```diff
@@ -104,13 +104,17 @@ export interface XpubPayload {
   readonly xpubType: XpubKeyType;
 }
 
-export interface BlockchainAccountPayload extends AccountPayload {
+export interface BasicBlockchainAccountPayload {
```

Comment:
Was this something we missed because the of the vuex store?

Reply:
Yes I guess, the type checker couldn't check it before.